docs/apache-airflow-providers-microsoft-azure/operators/sftp_to_wasb.rst

```diff
@@ -0,0 +1,65 @@
+
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Azure Blob Storage Transfer Operator
+====================================
+The Blob service stores text and binary data as objects in the cloud.
+The Blob service offers the following three resources: the storage account, containers, and blobs.
+Within your storage account, containers provide a way to organize sets of blobs.
+For more information about the service visit `Azure Blob Storage API documentation <https://docs.microsoft.com/en-us/rest/api/storageservices/blob-service-rest-api>`_.
+
+Before you begin
+^^^^^^^^^^^^^^^^
+Before using Blob Storage within Airflow you need to authenticate your account with Token, Login and Password.
+Please follow Azure
+`instructions <https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?tabs=azure-portal>`_
+to do it.
+
+Please, check `WASB <https://airflow.apache.org/docs/apache-airflow-providers-microsoft-azure/stable/connections/wasb.html>`_
+and `SFTP <https://airflow.apache.org/docs/apache-airflow-providers-sftp/stable/connections/sftp.html>`_
+for how to do such `connection <https://airflow.apache.org/docs/stable/howto/connection/index.html#editing-a-connection-with-the-ui>`_.
```

Comment:
For internal links in airflow documentation we don't use http links as it will break if the files you refer to are changed. You should use `ref`
https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

In general I think we can remove this paragraph as it's something relevant for all operators interacting with connections. If connection isn't setup the operator will not work.